content/learn/book/migration-guides/0.5-0.6/_index.md

```diff
@@ -61,9 +61,12 @@ impl Plugin for SomePlugin {
 }
 ```
 
+[`AppBuilder`]: https://docs.rs/bevy/0.5.0/bevy/app/struct.AppBuilder.html
+[`App`]: https://docs.rs/bevy/latest/bevy/app/struct.App.html
```

Comment:
This should probably point to 0.6 otherwise it might not point to the right thing once 0.6 isn't the latest